static/js/trie.js

```diff
@@ -0,0 +1,66 @@
+const Trie = () => {
```

Comment:
Should probably add a small comment here to describe what the trie tree is doing and what the data structure looks like after it's constructed (perhaps a reference to https://en.wikipedia.org/wiki/Trie).

Reply:
Done